spec/features/emojicom_rendering_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.feature "Emojicom rendering" do
+  context "home page" do
+    scenario "does not display emojicom widget" do
+      visit "/docs"
+
+      expect(page).not_to have_text "#emojicom-widget-inline"
```

Comment:
Is there a reason this one uses `have_text` while the others use `have_css`?

Reply:
Nope, good catch. And updated